ras_backstage/resources/collection_instrument/collection_instrument.py

```diff
@@ -25,7 +26,21 @@ def post(short_name, period):
         if not exercise:
             return make_response(jsonify({"message": "Collection exercise not found"}), 404)
 
-        collection_instrument_controller.upload_collection_instrument(survey['id'], exercise['id'],
-                                                                      request.files['file'])
+        upload_collection_instrument(survey['id'], exercise['id'], request.files['file'])
         logger.info('Successfully retrieved collection exercise details', shortname=short_name, period=period)
         return Response(status=201)
+
+    @staticmethod
+    def get(short_name, period):
+        logger.info('Retrieving collection instruments', short_name=short_name, period=period)
+
+        survey = survey_controller.get_survey_by_shortname(short_name)
+        exercises = collection_exercise_controller.get_collection_exercises_by_survey(survey['id'])
+        # Find the collection exercise for the given period
+        exercise = get_collection_exercise_by_period(exercises, period)
+        if not exercise:
+            return make_response(jsonify({"message": "Collection exercise not found"}), 404)
+        collection_instruments = get_collection_instruments_by_classifier(survey['id'], exercise['id'])
+
+        logger.info('Successfully retrieved collection instruments', short_name=short_name, period=period)
+        return make_response(jsonify(collection_instruments), 200)
```

Comment:
Currently this request is returning all `collection_instrument` information from the endpoint. We're only using the filename in the UI at the moment.

This could be fine though we just either need to filter down the information here or on the response-operations-ui PR